### PR TITLE
Basic user views for index & show

### LINKED ISSUE #7: Design bootstrap leaderboard

### PR DESCRIPTION
**Pull request by bayendor**
Closes #7

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
   helper_method :current_user
+  helper_method :user_signed_in?
+  helper_method :correct_user?
```
Comment (dalexj) [inline app/controllers/application_controller.rb]: should be `before_action`?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class UsersController < ApplicationController
+  before_filter :authenticate_user!
```
Comment (VikiAnn) [inline app/controllers/users_controller.rb]: I'm not sure if we need to authenticate the user for the index. Maybe we want new users to be able to preview the site a little before signing up?

**Comment by VikiAnn** [comment]
Looks great!  :+1:

**Comment by dalexj** [comment]
look goods

**Pull request merged**